pkg/extensions/extension_userprefs.go

```diff
@@ -43,6 +43,18 @@ func SetupUserPreferencesRoutes(config *config.Config, router *mux.Router, store
 	}
 }
 
+// ListTags godoc
+// @Summary Add bookmarks/ stars info
+// @Description Add bookmarks/ stars info
+// @Router 	/v2/_zot/ext/userprefs [put]
+// @Accept  json
+// @Produce json
+// @Param 	action	 	 query 	 string 		true	"specify action"
```

Comment:
Is there a way we can mention the supported values for `action`?
Also there is a `repo` Parameter for the existing `action`s, we should also mention that

Reply:
I made the changes